spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -244,9 +244,11 @@ public SparkContext createSparkContext() {
         new SparkConf()
             .setMaster(getProperty("master"))
             .setAppName(getProperty("spark.app.name"))
-            .setJars(jars)
             .set("spark.repl.class.uri", classServerUri);
 
+    if(0 < jars.length)
```

Comment:
We should have a consistent brackets style, as below. Same for goes for whitespaces.